backend/app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Comment;
+use App\Design;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+/**
+ * Class CommentController
+ * @package App\Http\Controllers
+ */
+class CommentController extends Controller
+{
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function create(Request $request)
+    {
+        $request->validate([
+            'design_id' => 'required|exists:design,id',
+            'comment' => 'required|min:5'
+        ]);
+
+        // Get the Design
+        $design = Design::find($request->input('design_id'));
+        $user = Auth::user();
+
+        // Create the Comment
+        $comment = new Comment();
+        $comment->user()->associate($user);
```

Comment:
Well there is https://laravel.com/docs/5.5/eloquent-resources for these things but we can always improvise later